pallets/subtensor/src/lib.rs

```diff
@@ -1277,6 +1278,83 @@ pub mod pallet {
             Self::do_set_weights(origin, netuid, dests, weights, version_key)
         }
 
+        // # Args:
+        // 	* `origin`: (<T as frame_system::Config>Origin):
+        // 		- The caller, a hotkey who wishes to set their weights.
+        //
+        // 	* `netuid` (u16):
+        // 		- The network uid we are setting these weights on.
+        //
+        // 	* `hotkey` (T::AccountId):
+        // 		- The hotkey associated with the operation and the calling coldkey.
+        //
+        // 	* `dests` (Vec<u16>):
+        // 		- The edge endpoint for the weight, i.e. j for w_ij.
+        //
+        // 	* 'weights' (Vec<u16>):
+        // 		- The u16 integer encoded weights. Interpreted as rational
+        // 		values in the range [0,1]. They must sum to in32::MAX.
+        //
+        // 	* 'version_key' ( u64 ):
+        // 		- The network version key to check if the validator is up to date.
+        //
+        // # Event:
```

Comment:
Is this meant to be an error on an event ?

Reply:
Meant to be an error.